app/controllers/SignInController.scala

```diff
@@ -0,0 +1,71 @@
+package controllers
+
+import scala.concurrent.duration._
+import scala.concurrent.{ ExecutionContext, Future }
+
+import com.mohiva.play.silhouette.api._
+import com.mohiva.play.silhouette.api.Authenticator.Implicits._
+import com.mohiva.play.silhouette.api.exceptions.ProviderException
+import com.mohiva.play.silhouette.api.util.{ Clock, Credentials }
+import com.mohiva.play.silhouette.impl.exceptions.IdentityNotFoundException
+import com.mohiva.play.silhouette.impl.providers._
+
+import javax.inject.Inject
+import net.ceedubs.ficus.Ficus._
+import play.api.Configuration
+import play.api.i18n.{ I18nSupport, Messages }
+import play.api.mvc.{ AbstractController, AnyContent, ControllerComponents, Request }
+
+import forms.SignInForm
+import models.services.UserService
+import utils.auth.DefaultEnv
+
+class SignInController @Inject() (
+  components: ControllerComponents,
```

Comment:
Should we try to be consistent with repeated dependency param names (`cc` vs. `components`)?